scripts/concurrent_task_utils.py

```diff
@@ -87,7 +90,7 @@ def _check_all_tasks(tasks):
             ))
 
         if task.exception:
-            ALL_ERRORS.append(task.exception)
+            ALL_ERRORS.append(task.stacktrace)
```

Comment:
@Hudda, should we add `self.name` in the log to give info about which linter test is failing?

Reply:
I don't think if that's necessary as stack trace is showing which file has errors. And even if we use `self.name` then `custom` or `third_party` as linter names isn't particularly useful.

Reply:
Yeah, I had realized that after adding the comments. Thanks for the clarification! :)